internal/client/client_test.go

```diff
@@ -123,7 +146,11 @@ func TestWrapAuth0ClientInfo(t *testing.T) {
 		t.Run(testCase.name, func(t *testing.T) {
 			testHandler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 				header := r.Header.Get("Auth0-Client")
-				assert.Equal(t, testCase.expected, header)
+				if testCase.expected == "" {
+					assert.NotEmpty(t, header)
+				} else {
+					assert.Equal(t, testCase.expected, header)
+				}
```

Comment:
Should this be reverted back now that we extracted the default client test to a separate t.Run?

Reply:
Absolutely! 🤦🏻